drivers/media/pci/intel/ipu-isys-video.c

```diff
@@ -30,12 +29,21 @@
 #include "ipu-fw-isys.h"
 #include "ipu-fw-com.h"
 
+#define IPU_IS_DEFAULT_FREQ	\
+	(IPU_IS_FREQ_RATIO_BASE * IPU_IS_FREQ_CTL_DEFAULT_RATIO)
+#define IPU6SE_IS_DEFAULT_FREQ	\
+	(IPU_IS_FREQ_RATIO_BASE * IPU6SE_IS_FREQ_CTL_DEFAULT_RATIO)
+
 /* use max resolution pixel rate by default */
 #define DEFAULT_PIXEL_RATE	(360000000ULL * 2 * 4 / 10)
 
 const struct ipu_isys_pixelformat ipu_isys_pfmts_be_soc[] = {
 	{V4L2_PIX_FMT_Y10, 16, 10, 0, MEDIA_BUS_FMT_Y10_1X10,
 	 IPU_FW_ISYS_FRAME_FORMAT_RAW16},
+	{V4L2_PIX_FMT_Y8I, 16, 16, 0, MEDIA_BUS_FMT_UYVY8_1X16,
+	 IPU_FW_ISYS_FRAME_FORMAT_UYVY},
+	{V4L2_PIX_FMT_Z16, 16, 16, 0, MEDIA_BUS_FMT_UYVY8_1X16,
+	 IPU_FW_ISYS_FRAME_FORMAT_UYVY},
```

Comment:
This is not good, as ipu_isys_get_pixelformat function looking for MEDIA_BUS_FMT code and returning pixel format so if user will request V4L2_PIX_FMT_UYVY the ipu6 will return the first it found matched by code MEDIA_BUS_FMT_UYVY8_1X16, V4L2_PIX_FMT_Y8I in that case, so i strongly suggest to rewrite ipu_isys_get_pixelformat  function or to move custom formats to end of the ipu_isys_pfmts_be_soc array.

Reply:
Thank you and we will consider this proposal in next release.